app/helpers/messages_helper.rb

```diff
@@ -0,0 +1,9 @@
+module MessagesHelper
+  def recipients_options
+    s = ''
+    User.all.each do |user|
+      s << "<option value='#{user.id}'>#{user.name}</option>"
```

Comment:
Rails has a built-in for that. Look at collection_select or collection_select_tag in the documentation